test/include/scenes.hpp

```diff
@@ -24,26 +25,31 @@ static rgl_entity_t makeEntity(rgl_mesh_t mesh= nullptr, rgl_scene_t scene=nullp
 	return entity;
 }
 
-static inline void setupBoxesAlongAxes(rgl_scene_t scene)
+static inline void spawnCubeOnScene(rgl_scene_t scene,
+                                    const Vec3f &position = {0, 0, 0},
+                                    const Vec3f &rotation = {0, 0, 0},
+                                    const Vec3f &scale = {1, 1, 1},
```

Comment:
It could be simplified by replacing these 3 parameters with `const Mat3x4f transform`

Reply:
I know that it is shorter this way, however, from the user's point of view 3 args are more convenient.
For example for parameterizations, quick changes of one vector and so one. In Your case, U have to create transform each time, and it will be a pain in the ass.

Reply:
`Mat3x4f` has a nice set of methods to create a transform. I see no pain here.
`TRS` method has the same parameres as yours, so it could be used in the parameterizations.

Reply:
Fair point. I forgot about TRS!
Jokes on me